### DISCUSSION TITLE
Help tabs localization OrchardCore

### DISCUSSION THREAD

**Discussion by Morgfrimen** [OrchardCMS/OrchardCore#11894] [Q&A] [answered]
Good afternoon, colleagues! I'm working with Orchard Core and I don't understand how the following elements can be localized, for example in Audit Trail Settings. I attach a screenshot.
![image](https://user-images.githubusercontent.com/44734894/175055140-51530aec-97db-45a1-a439-fad1cb5d174e.png)

OrchardCore generates tabs that are not clear how to localize.

**Comment by hishamco** [comment]
They should be localized if they are **STATIC** string. Let me have a look to AudiTrail module, one more thing be sure that are localized in your culture (Verify from Crowdin)

**Comment by hishamco** [comment]
I remember that tabs generated from placements which not support localization yet, @Piedone please confim if my words right or not

**Reply by Piedone** [reply]
Yep, as you found them :). I guess we need localization for them in a special way.

**Reply by hishamco** [reply]
I might prototype this in OCC, then I can push in OC. I remembered Seb or someone else mentioned them in Data Localization if I'm not wrong

**Comment by Morgfrimen** [comment]
@hishamco I didn't create my Audit Trail module. It is completely defaulted. The only thing I did was add the translation to the OrchardCore file.AuditTrail.po. And all static lines were localized. Except for tabs. I looked at the sources and couldn't find a place where they can be localized. It looks like they take their "system" name and that's it.

**Comment by hishamco** [comment]
>  I didn't create my Audit Trail module. It is completely defaulted

I knew ;) even default modules needs a translations .. I'm still checking them ..

**Comment by hishamco** [comment]
@Morgfrimen you can check AuditTrail [drivers](https://github.com/OrchardCMS/OrchardCore/tree/main/src/OrchardCore.Modules/OrchardCore.AuditTrail/Drivers), again I though that sort type of strings are dynamic by its nature so they aren't localized by default. So they should be covered in [Data Localization](https://github.com/OrchardCMS/OrchardCore/pull/4466)

**Comment by ns8482e** [comment, marked as answer]
Tab generated from placement rule so they can’t be localized by default  

however tabs are shapes you can override them to localize text

**Comment by Morgfrimen** [comment]
thank you very much for the answers!
in the near future I will try to fix it based on your answers. I will inform you about the result
! thank you again!

**Comment by Morgfrimen** [comment]
Made a decision to generate a new shapes. I'll try to go this way, since I don't see any other way. Thank you for the answers!

**Reply by hishamco** [reply]
This is a workaround but this should be supported in upcoming future
